tests/integration/test_opensearch.py

```diff
@@ -585,7 +586,10 @@ def test_endpoint_strategy_port_singleton_cluster(self, monkeypatch):
 
         parts = cluster_0.url.split(":")
         assert parts[0] == "http"
-        assert parts[1] == "//localhost"
+        if in_docker():
+            assert parts[1] == "//0.0.0.0"
+        else:
+            assert parts[1] in ("//localhost", "//127.0.0.1")
```

Comment:
In essence here it should always contain the `EDGE_BIND_HOST` (no matter if it's in docker or not), right?

Reply:
Yes, but I wanted to have it a bit more explicit so that the tests documents the behaviour. I can change it to `EDGE_BIND_HOST` though.

Reply:
I think I would prefer the `EDGE_BIND_HOST` here, as it states the actual behavior instead of implicit effects.